transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/Carousel.kt

```diff
@@ -0,0 +1,8 @@
+package com.genesys.cloud.messenger.transport.core
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Carousel(
```

Comment:
Add KDoc here as well.

Also, are we planning to expose Carousel object to UI layer?

Reply:
I believe we decided to remove this class completely.